pkgs/by-name/wa/walksnail-osd-tool/package.nix

```
@@ -0,0 +1,78 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, makeWrapper
+, wrapGAppsHook3
+, makeDesktopItem
+, copyDesktopItems
+, rustPlatform
+, cargo
+, cmake
+, pkg-config
+, libGL
+, openssl
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "walksnail-osd-tool";
+  version = "v0.3.0"; # don't forget to update the make-build-reproducible.patch
+
+  src = fetchFromGitHub {
+    owner = "avsaase";
+    repo = pname;
+    rev = "1044026bab773386fe96e20b544f60ad73d24e3f";
+    hash = "sha256-xCrshFRsM4qUF4TffZiriNaplkjif/LeFOwLxoqtwsY=";
+  };
+
+  patches = [
+    ./make-build-reproducible.patch
+  ];
+
+  cargoLock = {
+    lockFile = ./Cargo.lock;
+    outputHashes = {
+      "ffprobe-0.3.3" = "sha256-T6ht3ZS68Hsfw+t9uGO+ZveZJtvEH3nCd+N3vh8g5HM=";
+      "poll-promise-0.2.0" = "sha256-IrV0asFTu2P/FW60ft0PphFzfmkR08M/YBwUXuGVRXk=";
+    };
+  };
+
+  nativeBuildInputs = [
+    cargo
+    cmake
+    copyDesktopItems
+    openssl.dev
+    pkg-config
+    rustPlatform.cargoSetupHook
+    wrapGAppsHook3
+  ];
+  buildInputs = [
+    openssl
+  ];
+  OPENSSL_NO_VENDOR = 1;
+
+  postFixup =
+    let
+      rpath = lib.makeLibraryPath [ libGL ];
+    in
+    ''
+      patchelf --add-rpath ${rpath} $out/bin/.${pname}-wrapped
+    '';
```

Could be simplified.
```suggestion
  postFixup = ''
    patchelf $out/bin/.${pname}-wrapped \
      --add-rpath ${lib.makeLibraryPath [ libGL ]} 
  '';
```